.fantomasignore

```diff
@@ -0,0 +1 @@
+src/packet.references 
```

Comment:
I'm not sure why this is needed (typo anyway, shouldn't this be `paket`?)

Reply:
I didn't look into it much when I copied the code from the dark repo. It was in the same directory as the other files. I think It should be in the same directory as paket.dependencies.